runtime/configurationparser/ConfigurationPrinter.cpp

```diff
@@ -216,6 +216,66 @@ extern "C" void* getStderr(void) {
   return stderr;
 }
 
+struct MatchLog {
+  enum {
+    SUCCESS=0, FUNCTION, FAIL
+  } kind;
+
+  char *function;
+  char *debugName;
+  void *result;
+  std::vector<void *> args;
+
+  char *pattern;
+  void *subject;
+  char *sort;
+};
+
+static std::vector<MatchLog> matchLog;
+
+void resetMatchReason(void) {
+  matchLog.clear();
+}
+
+MatchLog *getMatchLog(void) {
+  return &matchLog[0];
+}
+
+void **getMatchFnArgs(MatchLog *log) {
+  return &log->args[0];;;;
```

Comment:
I'm curious: Why so many semicolons?

Reply:
My editor sometimes does this by mistake and I didn't spot this one.